components/01-atoms/typography/text/_yds-text.scss

```diff
@@ -6,6 +6,95 @@ p {
   p:last-child {
     margin-bottom: 0;
   }
+
+  // Code blocks.
+  pre code {
+    display: block;
+    padding: 1em;
+
+    // from highlightjs Monokai Sublime
+    // @see highlight.js/styles/monokai-sublime.css
+
+    /*
+    Monokai Sublime style. Derived from Monokai by noformnocontent http://nn.mit-license.org/
+    */
+
+    &.hljs {
+      background: #23241f;
+      color: #f8f8f2;
+      overflow-x: auto;
+    }
```

Comment:
@laura-johnson I'd like to see all of these hex colors replaced with CSS variables or color tokens, if possible. Adding the colors as tokens is likely more involved than what's needed here, so maybe we could define the colors as CSS variables at the top of this file. 

Something like: 
```
// Code highlighting color variables

:root {
--color-hljs-text: #f8f8f2;
--color-hljs-background: #23241f;
}
```

And then, this part: 
```
   &.hljs {
      background: #23241f;
      color: #f8f8f2;
      overflow-x: auto;
    }
```

Would turn into: 

```
   &.hljs {
      background: var(--color-hljs-text);
      color: var(--color-hljs-background);
      overflow-x: auto;
    }
```

Please let me know if you'd rather tag me in to make these hex to CSS variable changes. 👍